pact.cabal

```diff
@@ -164,10 +164,6 @@ library
     , vector >= 0.11.0.0 && < 0.13
     , vector-algorithms >= 0.7
     , vector-space >= 0.10.4 && < 0.17
-  if flag(ghc-flags)
```

Comment:
Why was this removed?

Reply:
Stale build tool-related entry. Sam's `hsinspect` stuff.

Reply:
Ah okay :+1: